src/modules/commander/commander_params.c

```diff
@@ -339,6 +339,20 @@ PARAM_DEFINE_INT32(COM_ARM_SWISBTN, 0);
  */
 PARAM_DEFINE_INT32(COM_LOW_BAT_ACT, 0);
 
+/**
+ * Predictive battery failsafe
+ *
+ * If this parameter is enabled, the battery failsafe action defined by COM_LOW_BAT_ACT will trigger predictively
+ * based on the estimated remaining state of charge after an RTL.
+ *
+ * @group Commander
+ * @value 0 Disabled
+ * @value 1 Enabled
+ * @decimal 0
+ * @increment 1
+ */
+PARAM_DEFINE_INT32(COM_PRED_BAT_FS, 0);
```

Comment:
Suggest default value 1/enabled, so the default behavior will not change.

Reply:
Good point. I set the default value to 0 since that is what we will want to use most of the time, but keeping the default behaviour is probably more worth.